### ISSUE TITLE
Fails to pip install

### ISSUE THREAD

**Issue by richardasaurus** [wwoods/keras_pickle_wrapper#3]
Python 3.6

```bash
>> pip install keras-pickle-wrapper

Collecting keras_pickle_wrapper
  Downloading keras-pickle-wrapper-1.0.3.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/tmp/pip-build-v58sxrt0/keras-pickle-wrapper/setup.py", line 5, in <module>
        'rt') as f:
    FileNotFoundError: [Errno 2] No such file or directory: '/private/tmp/pip-build-v58sxrt0/keras-pickle-wrapper/README.md'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/tmp/pip-build-v58sxrt0/keras-pickle-wrapper/
```

**Comment by richardasaurus** [comment]
<img width="469" alt="screen shot 2018-02-16 at 19 30 44" src="https://user-images.githubusercontent.com/2279659/36325507-e9f66130-134f-11e8-87eb-f07c0553e402.png">

that file is missing from the tar file on pypi

**Comment by wwoods** [comment]
Hi @richardasaurus , thanks for calling this to my attention.  It's been fixed in 1.0.4.

Cheers,
Walt
